install-npm.js

```diff
@@ -4,8 +4,14 @@ var npmModule
   , path = require('path')
   ;
 
-if (process.env['npm_execpath'] && process.env['npm_execpath'].match(/\/node_modules\/npm\/bin\/npm-cli\.js$/)) {
-  npmModule = require(path.resolve(process.env['npm_execpath'], '..', '..'));
+if (process.env['npm_execpath']) {
+  if (process.platform === 'win32') {
+    process.env['npm_execpath'] = process.env['npm_execpath'].replace(/\\/g, '/');
```

Comment:
I think we can do better than to mangle environment variables with something that will break downstream functionality.

Reply:
Done, no more changes to env vars.